packages/stark-core/assets/css/pre-load-style.css

```diff
@@ -0,0 +1,93 @@
+/* Hide the app / loader / browser warning until it is determined if JS is enabled and the if the browser is compatible  */
+.stark-app,
+.stark-loading,
+.stark-browser-warning {
+  display: none;
+}
+
+.stark-browser-outdated .stark-browser-warning {
+  display: block;
+}
+
+.stark-browser-ok .stark-app {
+  display: flex;
+}
+
+.stark-browser-ok .stark-loading {
+  display: block;
+}
+
+.stark-preload-ui-page h1 {
```

Comment:
Can we rename the `stark-preload-ui-page`? I think that "ui-page" is not 100% accurate since it is not a page itself...